### DISCUSSION TITLE
Cache and loading time

### DISCUSSION THREAD

**Discussion by yuhengh** [GrandOrgue/grandorgue#1006] [General]
I'm new to GrandOrgue and using it with Mac Catalina on a 2013 Macbook Pro. Thanks for the amazing work!

I notice the loading time is much faster with cache (~10 seconds) vs without cache (~5 mins). When I switch to another organ, the cache seems to be disabled and it takes 5 mins to load instead of 10 seconds. Even restarting GrandOrgue disable cache sometimes.  Is it working as intended? How does cache work exactly?

**Comment by willeke1234** [comment]
I'm on a 2012 Mac mini with macOS Catalina. I don't know if the cache is working as intended but what I do know:
Each organ has its own cache.
macOS may delete cache files in the Caches folder if it needs disk space.
macOS and the disk are also caching. Reloading the same organ can be much faster, with or without cache.

**Reply by yuhengh** [reply]
Yes, I set my cache folder to a 1T external hard drive so all the cache files should be there and there shouldn't be space issue. I'm wondering if switching to a different organ (with cache) is much slower is expected behavior and if there's space for improvement.
